enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -62,6 +69,7 @@ var (
 	podmanRuntime       = flag.String("executor.podman.runtime", "", "Enables running podman with other runtimes, like gVisor (runsc).")
 	podmanStorageDriver = flag.String("executor.podman.storage_driver", "overlay", "The podman storage driver to use.")
 	podmanEnableStats   = flag.Bool("executor.podman.enable_stats", true, "Whether to enable cgroup-based podman stats.")
+	enableExecServer    = flag.Bool("executor.podman.enable_exec_server", false, "Whether to use an execution server inside each podman container for executing commands, rather than using 'podman exec'. Applies only to actions with runner recycling enabled.")
```

Comment:
From the code it seems like this applies to all actions and not just those with runner recycling enabled?

Also why does it matter if runner recycling is enabled? If this is faster, should we always use it?

Reply:
In the recycling case, we are replacing the `podman exec` command with a gRPC call to the exec server, which is faster. In the non-recycling case, we are using `podman run` to create the container and execute the command in a single podman command. I wouldn't expect the exec server to speed up `podman run`, since most of the overhead in `podman run` is in creating and starting the container.